libbeat/scripts/generate_fields_docs.py

```diff
@@ -73,9 +73,11 @@ def document_field(output, field, field_path):
             output.write("{}\n\n".format("Object is not enabled."))
 
```

Comment:
Would it work the same if you just moved the original `output.write` line to here?

Reply:
Yes, that works! Thanks for helping me simplify the logic. I'm good at getting things to work...but not so good at keeping it simple and elegant. :-)